src/operations/operation.ts

```diff
@@ -49,7 +49,6 @@ const kSession = Symbol('session');
  */
 export abstract class AbstractOperation<TResult = any> {
   ns!: MongoDBNamespace;
-  cmd!: Document;
```

Comment:
removed because this no longer exists?

Reply:
Kinda, the `xx!` properties on this class are tech debt from the rapid TS conversion, we couldn't sort out which subclasses actually defined these and which didn't. Specifically cmd what only being attached by the  `RenameOperation` so we don't need it defined on this super class. But also Rename is somewhat special in that most of our commands are build in the exec function, not in the constructor, so even Rename doesn't define this property anymore.